webservices/common/models/itemized.py

```diff
@@ -94,9 +97,8 @@ class ScheduleA(BaseItemized):
     contributor_middle_name = db.Column('contbr_m_nm', db.String)
     contributor_last_name = db.Column('contbr_nm_last', db.String)
     contributor_suffix = db.Column('contbr_suffix', db.String)
-    # Street address omitted per FEC policy
-    # contributor_street_1 = db.Column('contbr_st1', db.String)
-    # contributor_street_2 = db.Column('contbr_st2', db.String)
+    contributor_street_1 = db.Column('contbr_st1', db.String)
+    contributor_street_2 = db.Column('contbr_st2', db.String)
```

Comment:
Are we sure these need to go back in?  They were omitted previously at FEC's request.